### ISSUE TITLE
Ajouter un condensateur en parallèle du relai

### ISSUE THREAD

**Issue by gagath** [haum/PCBlastifieuse#4]
Pour ne pas que le relai tire trop lors de son activation et perturbe l’ADC.

**Comment by jblb** [comment]
pas en parallèle du relais mais en parallèle de l'alim du relais

**Comment by gagath** [comment]
Oui, entre le +5 et le GND du relai ; par contre on part sur quelle valeur ? On teste avec une capacité moyenne pour voir si ça suffit ou on part dans des vrais calculs de dimensionnement ?

**Comment by jblb** [comment]
220µF ou 470µF suivant ce que l'on trouve!

**Comment by gagath** [comment]
De toute façon il faudrait également que l'on se fasse notre propre PCB pour la carte relai de la machine ; on pourrait directement intégrer ce condensateur sur la carte plutôt que de le brancher en externe.

**Comment by neomilium** [comment]
Refaire ce qui a été fait alors que c'est déjà fait? Là, il ne manque qu'un
condensateur sur le bornier et c'est bon, non ?

Le 20 novembre 2015 08:51, Romain Porte notifications@github.com a écrit :

> De toute façon il faudrait également que l'on se fasse notre propre PCB
> pour la carte relai de la machine ; on pourrait directement intégrer ce
> condensateur sur la carte plutôt que de le brancher en externe.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/haum/PCBlastifieuse/issues/4#issuecomment-158314077.

## 

Romuald

**Comment by gagath** [comment]
C'était dans l'optique de te rendre ta carte relai que tu voulais récupérer à la base surtout mais si tu n'y tiens pas plus que ça désormais oui on le met en externe !

**Comment by neomilium** [comment]
Elle appartient au HAUM désormais. C'est vrai qu'elle très pratique pour du
prototypage rapide donc faire un PCB n'est pas une mauvaise idée en soit,
mais n'est pas une obligation.

Le 20 novembre 2015 16:20, Romain Porte notifications@github.com a écrit :

> C'était dans l'optique de te rendre ta carte relai que tu voulais
> récupérer à la base surtout mais si tu n'y tiens pas plus que ça désormais
> oui on le met en externe !
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/haum/PCBlastifieuse/issues/4#issuecomment-158430070.

## 

Romuald

**Comment by gagath** [comment]
Tout à fait.

Je propose donc de monter le condensateur pour vérifier l'amélioration au niveau de la mesure et dans un futur plus lointain se refaire une carte si on trouve un relai adéquat.

**Comment by neomilium** [comment]
+1

Le 20 novembre 2015 19:25, Romain Porte notifications@github.com a écrit :

> Tout à fait.
> 
> Je propose donc de monter le condensateur pour vérifier l'amélioration au
> niveau de la mesure et dans un futur plus lointain se refaire une carte si
> on trouve un relai adéquat.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/haum/PCBlastifieuse/issues/4#issuecomment-158482500.

## 

Romuald
